dht/dht.go

```diff
@@ -980,11 +980,13 @@ func bootstrapAddrs(nodeAddrs []string) (addrs []*net.UDPAddr, err error) {
 		}
 	}
 	for _, addrStr := range bootstrapNodes {
-		udpAddr, err := net.ResolveUDPAddr("udp4", addrStr)
-		if err != nil {
-			continue
+		if addrStr != "" {
```

Comment:
I don't have a name for this, but it's like passing the buck on a sentinel value. You've created a new special case which is addrStr=="". Maybe it could mean something. Maybe not. It's not our problem unless we care about the errors we're getting, which we don't. What do you think of that?